### ISSUE TITLE
Make heading level customisable

### ISSUE THREAD

**Issue by atimmer** [Yoast/YoastSEO.js#382]
After #374, we have improved the heading structure for WordPress. Ultimately YoastSEO.js is a cross platform project so we have to work on other platforms as well.

A simple example of this is https://yoast.com/research/real-time-content-analysis/. There we really need h2s.

**Comment by afercia** [comment]
Definitely yes, and maybe we should consider this also for PHP. Things may always change in the future and there should be an easy way to change the HTML output, especially about elements that convey important semantics like headings. See for example:
https://github.com/Yoast/wordpress-seo/blob/3.1.2/admin/metabox/class-metabox.php#L335

**Comment by omarreiss** [comment]
Let's have it come up again in the wild and then fix.
